libqtile/core/manager.py

```diff
@@ -92,6 +92,7 @@ def __init__(
 
         self.windows_map: Dict[int, base.WindowType] = {}
         self.widgets_map: Dict[str, _Widget] = {}
+        self._widgets: List[_Widget] = []
```

Comment:
I'd have a preference not to create a new variable. Instead, a better outcome would be to make sure `widgets_map` includes everything. This would have the added benefit of making every widget more easily available via IPC.